ArduCopter/xrce-client.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Copter.h"
```

Comment:
This should probably be written as a library instead

Reply:
Okay will do so